RecoTracker/GeometryESProducer/python/TrackerRecoGeometryESProducer_cfi.py

```diff
@@ -1,5 +1,9 @@
 import FWCore.ParameterSet.Config as cms
 
-TrackerRecoGeometryESProducer = cms.ESProducer("TrackerRecoGeometryESProducer")
+TrackerRecoGeometryESProducer = cms.ESProducer("TrackerRecoGeometryESProducer",
+  usePhase2Stacks = cms.bool(False)
+)
```

Comment:
If the `fillDescriptions()` would be set to generate a cfi, and this file would use the generated one, this change would not be needed.

Reply:
To make sure I understand correctly, to do that I would change

`descriptions.addDefault(desc);` to `descriptions.addDefault("TrackerRecoGeometryESProducer",desc);` in `TrackerRecoGeometryESProducer.cc`? And then the `toModify` statement below would have to be moved someplace else to do the switch?

Reply:
Assuming the (currently no-op) `toModify()` is really needed in the future, the recommended pattern is to do in `fillDescriptions()` something along
```c++
descriptions.add("trackerRecoGeometryESProducerDefault", desc);
```
(note that the naming convention is to start module labels and cfi/cff files with lower case even if it is widely violated), and then in this file
```python
from RecoTracker.GeometryESProducers.trackerRecoGeometryESProducerDefault_cfi import trackerRecoGeometryESProducerDefault as _trackerRecoGeometryESProducerDefault
TrackerRecoGeometryESProducer = _trackerRecoGeometryESProducerDefault.clone()
```
Then the default values will be generated, there is a customization point for the modifier, and it is transparent to all current users of `TrackerRecoGeometryESProducer_cfi`.

Reply:
Thinking about it, I am not sure it will be needed in the future once the full configuration update to include VHs in RECO is integrated. So I would for now keep it as is as a convenient switch and fully replace this file with one generated by fillDescriptions at that point.